src/pages/video/[videoId].tsx

```diff
@@ -1,29 +1,58 @@
-import { useRouter } from "next/router"; //using useRouter hook so we can dinamically route to video based on ID
-import React from "react";
+import { useState, useEffect } from 'react'
+import { useRouter } from "next/router";
 import { Profile } from "src/components";
-import david from "public/davidIcon.jpeg";
+import { getVideos } from 'lib/videos';
 
+const creatorData = getVideos('creatorOne');
 const Video = () => {
 
   const router = useRouter();
-  console.log({ router });
+
+  const [isSubscribe, setIsSubscribe] = useState(false);
+
+  useEffect(() => {
+    const handleSubscribe = async () => {
+
+      try {
+        const sub = true;
```

Comment:
Don't understand

Reply:
This is work in progress. This function should be finished after we solve Stripe authentication issue.
it should be : const sub = {function that returns boolean is it user subscribed or not}.

Reply:
Same as i mentioned last time - work in progress shouldn't be in the main
maybe in a separate branch for future work 
this is creating tech debt